### ISSUE TITLE
Evaluation for depth estimation & Directory confusion

### ISSUE THREAD

**Issue by Shuang1997** [eugenelyj/blinkvision_benchmark#5]
Hi, thanks for your efforts in building such a big dataset. I have downloaded the data and I have two questions:

(1) In the benchmark website, there are only flow and point tracking now. Would you provide the benchmark for depth estimation in the near future? I am asking because there is no GT data in the test set.

(2) In the training set, there are four folders: indoor_train, indoor_train_event, outdoor_train, outdoor_train_event. What are the differences between indoor_train & indoor_train_event and outdoor_train & outdoor_train_event? Both have the same sequences names, while both have the event_left folder (seems the event data is different)? Do you mean that the *_train is for both frame-based and event-based methods while *_train_event is for event-only methods?

Looking forward to hearing from you soon!

**Comment by eugenelyj** [comment]
@Shuang1997 Hello,
(1) We are considering adding the benchmark of depth/normal, but we are still thinking about how to design it and need to communicate with people in the field.
(2) Actually, the indoor_train and indoor_train_event can be merged. The indoor_train_event, If I remember correctly, contains the correct corresponding event data. The different folder name is to avoid data overwrite because we previously rendered different resolution data but only used one of them in the end. We recommend use the provided dataloader so you can ignore the details.

**Comment by Shuang1997** [comment]
@eugenelyj Hi, thanks for your reply. 
(1) How long will depth benchmark roughly take? Hopefully we can use your dataset to evaluate our depth estimation method.
(2) I am still confused (why can they be merged?). indoor_train_event and outdoor_train_event only the following data (only GT forward flow is given):
![Image](https://github.com/user-attachments/assets/dd425751-a9f2-40ed-a0e6-cb3df8135dec)
while indoor_train have more data (GT depth, formal...):
![Image](https://github.com/user-attachments/assets/ca210ba0-b53d-4ce1-bb3a-63bd0a470e41)
Which one did you use in the end? In the provided data loader, only the indoor_train and outdoor_train are loaded while *_train_event are not loaded.

**Comment by eugenelyj** [comment]
@Shuang1997 Hello,
(1) What is your setting, event-only depth estimation conditioned on single slice of event stream, or other? Could you provide some detailed description and also provide some common metrics that are used in your setting, I would consider adding it next week.
(2) Sorry that we miss the event loading in the provided train dataloader, I would add it soon, then you can refer to it.

**Comment by Shuang1997** [comment]
@eugenelyj Hi, 
(1) Our method uses both RGB images and events to predicts monocular depth at the timestamps of every RGB image. For metrics, the common ones are absolute relative error (AbsREL), root mean squared error (RMSE), average log10 error, and the threshold accuracy. Please refer to [ZoeDepth: Zero-shot Transfer by Combining Relative and Metric Depth](https://arxiv.org/abs/2302.12288) for full details. Note that affine-invariant scaling is needed before computing the above metrics due to the scale ambiguity of monocular depth estimation (relative depth, no absolute scale).
(2) Thank you for that! Please let me know when you add it.

**Comment by eugenelyj** [comment]
Hi @Shuang1997 
(1) The depth benchmark is still in progress. I think it will take another week. I will notify you when it is done.
(2) Added now.

**Comment by eugenelyj** [comment]
The depth benchmark has been added in August.
